Bio/File.py

```diff
@@ -183,19 +191,24 @@ def peekline(self):
         return line
 
     def tell(self):
+        """Return the current position of the file read/write pointer within the File."""
         return self._handle.tell() - sum(len(line) for line in self._saved)
 
     def seek(self, *args):
+        """Set the current position at the offset specified."""
         self._saved = []
         self._handle.seek(*args)
 
     def __getattr__(self, attr):
+        """Return File attribute."""
         return getattr(self._handle, attr)
 
     def __enter__(self):
+        """Open File handle with WITH statement."""
```

Comment:
See discussion on #1170, not everyone agrees about strictly enforcing ``D401: First line should be in imperative mood; try rephrasing``, but since we're adding missing docstrings here we might as well try to do this.